src/pages/contact.js

```diff
@@ -0,0 +1,55 @@
+//Gerardo Colon
+//NOGD internship
+//Contact page 
+//Last edited: 9/13/2023
+
+import * as React from 'react'
+import Layout from '../components/layout'
+import { Link } from 'gatsby'
+
+const ContactPage = () => {
```

Comment:
make this a named function and not an anonymous function inside a named variable